sites/hashai/scripts/sync-algolia-index.ts

```diff
@@ -156,7 +156,7 @@ const main = async () => {
     await syncAlgoliaIndex();
     console.log("Algolia Indexes Updated.");
   } catch (error) {
-    throw new Error(`Algolia Indexing Failed: ${error}`);
+    throw new Error(`Algolia Indexing Failed: ${error as string}`);
```

Comment:
How about this? If `error` is a string, it remains a string. If it's some random other thing we might get more useful logs. And avoids a cast
```suggestion
    throw new Error(`Algolia Indexing Failed: ${JSON.stringify(error)}`);
```